src/dmd/dtoh.d

```diff
@@ -652,7 +652,8 @@ public:
 
                 case EnumKind.String, EnumKind.Enum:
                     buf.writestring("static ");
-                    writeEnumTypeName(type);
+                    auto target = determineEnumType(type);
+                    target.accept(this);
```

Comment:
Why is it not a single statement like the other calls ?

Reply:
Because `target` is used a few lines below.